src/test/regress/sql/citus_local_tables_queries_mx.sql

```diff
@@ -0,0 +1,615 @@
+\set VERBOSITY terse
```

Comment:
This seems to be very similar to `citus_local_tables_queries.sql`. It will be very hard to maintain these two files. Can we have a single test file and call it both from a mx cluster and non-mx cluster?

If the output is different, we can add an alternative output.

With a single file we can easily see the effect of adding a new test in both mx and non-mx mode.

We can possibly do the same thing for `citus_local_tables` vs `citus_local_tables_mx`?